mmgen/models/architectures/biggan/modules.py

```diff
@@ -432,3 +432,316 @@ def forward(self, x):
             x0 = self.downsample(x0)
         x1 = self.forward_sc(x)
         return x0 + x1
+
+
+@MODULES.register_module()
+class BigGANDeepGenResBlock(nn.Module):
+    """Residual block used in BigGAN-Deep's generator.
+
+    Args:
+        in_channels (int): The channel number of the input feature map.
+        out_channels (int): The channel number of the output feature map.
+        dim_after_concat (int): The channel number of the noise concatenated
+            with the class vector.
+        conv_cfg (dict, optional): Config for the convolution module used in
+            this residual block. Defaults to dict(type='Conv2d').
+        act_cfg (dict, optional): Config for the activation layer. Defaults to
+            dict(type='ReLU').
+        upsample_cfg (dict, optional): Config for the upsampling operation.
+            Defaults to dict(type='nearest', scale_factor=2).
+        sn_eps (float, optional): Epsilon value for spectral normalization.
+            Defaults to 1e-6.
+        bn_eps (float, optional): Epsilon value for batch normalization.
+            Defaults to 1e-5.
+        with_spectral_norm (bool, optional): Whether to use spectral
+            normalization in this block. Defaults to True.
+        input_is_label (bool, optional): Whether the input of BNs' linear layer
+            is raw label. Defaults to False.
```

Comment:
xxx is raw labels **instead of one-hot labels**. This makes the definition of raw labels clear.